### PR TITLE
FIX: nullable reference for all msgs

### PR DESCRIPTION
**Pull request by vincentysc**

### CODE REVIEW

File: usecase/parser/msg.go [hunk 1]
```diff
@@ -176,14 +177,14 @@ func ParseMsgMultiSend(
 ) ([]command.Command, []string) {
 	rawInputs, _ := parserParams.Msg["inputs"].([]interface{})
```
Comment (davcrypto) [inline usecase/parser/msg.go:178]: no need for the invalid data check here?

File: usecase/parser/msg.go [hunk 2]
```diff
@@ -176,14 +177,14 @@ func ParseMsgMultiSend(
...
 	inputs := make([]model.MsgMultiSendInput, 0, len(rawInputs))
-	addresses := make([]string, 0, len(rawInputs))
+	possibleSignerAddresses := make([]string, 0, len(rawInputs))
 	for _, rawInput := range rawInputs {
 		input, _ := rawInput.(map[string]interface{})
 		inputs = append(inputs, model.MsgMultiSendInput{
 			Address: input["address"].(string),
 			Amount:  tmcosmosutils.MustNewCoinsFromAmountInterface(input["coins"].([]interface{})),
 		})
-		addresses = append(addresses, input["address"].(string))
```
Comment (davcrypto) [inline usecase/parser/msg.go:186]: no need for the invalid data check here?

Comment (vincentysc) [reply usecase/parser/msg.go:186]: oh, I add it back now.

File: usecase/parser/msg.go [hunk 3]
```diff
@@ -1289,22 +1494,30 @@ func ParseMsgRevoke(
 			RawMsgRevoke: rawMsg,
 		}
 
+		// Getting possible signer address from Msg
+		var possibleSignerAddresses []string
+		possibleSignerAddresses = append(possibleSignerAddresses, revokeParams.Granter)
```
Comment (davcrypto) [inline usecase/parser/msg.go:1503]: may i know where do we assign `Granter` to `revokeParams`?

Comment (vincentysc) [reply usecase/parser/msg.go:1503]: The `RawMsgRevoke` type got `Granter`
https://github.com/crypto-com/chain-indexing/blob/e6d137d63ffe8b75710ea64b356798e1219b5a3b/usecase/parser/msg.go#L1497

Comment (davcrypto) [reply usecase/parser/msg.go:1503]: looks like `revokeParams. Granter` must be `nil`?

Comment (vincentysc) [reply usecase/parser/msg.go:1503]: so using `revokeParams.RawMsgRevoke.Granter` instead, right?

Comment (davcrypto) [reply usecase/parser/msg.go]: `revokeParams.RawMsgRevoke.Granter` make more sense. And i saw other parsers might have similar situation, can you help to double check?

Comment (vincentysc) [reply usecase/parser/msg.go]: yea, sure

**Review by davcrypto — APPROVED** [review]

**Pull request merged**